### PR TITLE
Add option to WebP encoder to ignore animation

### PR DESCRIPTION
**Pull request by SpikeViper**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp/pulls) open
- [x] I have verified that I am following the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [?] I have provided test coverage for my change (where applicable)
   - Would be glad to, not really sure how to write such a test

### Description
Details at: https://github.com/SixLabors/ImageSharp/discussions/2697

TLDR; While the new ability to encode animated WebPs is amazing, there are cases in which one would want to encode both an animated and static version of the image without decoding multiple times. A simple flag is sufficient to make this possible.

### CODE REVIEW

**Comment by SpikeViper** [comment]
Closing due to being deemed "encoder specific hack".
